include/roboteam_ai/interface/widgets/TreeVisualizerWidget.h

```diff
@@ -23,13 +23,13 @@ class TreeVisualizerWidget : public QTreeWidget {
     QColor getColorForStatus(bt::Node::Status status);
     void addRootItem(bt::Node::Ptr parent, QTreeWidgetItem* QParent);
     std::map<QTreeWidgetItem *, bt::Node::Ptr> treeItemMapping;
-    bool hasCorrectTree = false;
+    static bool hasCorrectTree;
```

Comment:
Okay I do not recommend doing things like this. Did you know we have two instances of treevisualizerwidget?